src/api/eventsApi.tsx

```diff
@@ -0,0 +1,13 @@
+import { spWebContext } from "api/SPWebContext";
+import { useMutation } from "@tanstack/react-query";
+
+export interface EventLogItem {
+  requestId: number;
+  Title: string;
+}
+
+export const useAddEvent = () => {
+  return useMutation(["addEvent"], async (item: EventLogItem) => {
+    return spWebContext.web.lists.getByTitle("events").items.add(item);
```

Comment:
Make sure we set up index(es) on these so that we can query things easier as the log grows.